Sources/BaggageLogging/BaggageContextLogging.swift

```diff
@@ -0,0 +1,125 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Baggage Context open source project
+//
+// Copyright (c) 2020 Moritz Lang and the Swift Baggage Context project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Baggage
+import Logging
+
+/// A `BaggageContextLogging` purpose is to be adopted by frameworks which already provide a "FrameworkContext",
+/// and to such frameworks to pass their context as `HasBaggageContext`.
+public protocol BaggageContextLogging: BaggageContextCarrier {
```

Comment:
I like `BaggageContextCarrier` as the base protocol name 👍 How about `LoggingBaggageContextCarrier` for this protocol? A bit lengthy, but at least descriptive.

Reply:
`BaggageContextCarrier` yeah maybe that's good hm... I'll try that.

The longer one is a bit too long, that's a log of typing we'd ask people to do hm

Reply:
Hmmm yeah I guess that's "correct" but very long :-(

We can try that for now, thanks